### ISSUE TITLE
remark-align does not parse blocks

### ISSUE THREAD

**Issue by artragis** [zestedesavoir/zmarkdown#97]
zmd allows blocks to be aligned and as remark-caption is just inlineTokenizer we don't really parse what we want. For example : 

```markdown
->
- 1
- 2
- 3
->
```

is parsed as 
```
root[2] (1:1-5:3, 0-17)
├─ paragraph[1] (1:1-1:3, 0-2)
│  └─ text: "->" (1:1-1:3, 0-2)
└─ list[3] (2:1-5:3, 3-17) [ordered=false][loose=false]
   ├─ listItem[1] (2:1-2:4, 3-6) [loose=false]
   │  └─ paragraph[1] (2:3-2:4, 5-6)
   │     └─ text: "1" (2:3-2:4, 5-6)
   ├─ listItem[1] (3:1-3:4, 7-10) [loose=false]
   │  └─ paragraph[1] (3:3-3:4, 9-10)
   │     └─ text: "2" (3:3-3:4, 9-10)
   └─ listItem[1] (4:1-5:3, 11-17) [loose=false]
      └─ paragraph[1] (4:3-5:3, 13-17)
         └─ text: "3\n->" (4:3-5:3, 13-17)
```

Common cases in tutorial are things like : 

```
->
!(https://www.youtube.com/watch?v=i190q5WzRHo)
Video: my caption
<-
```

or table alignement, and making lists central.

So this is important for tutorial parsing.

**Comment by AmarOk1412** [comment]
I'll do it this week. Thanks for reporting!

**Comment by AmarOk1412** [comment]
By the way:
```
->a->
->b<-
```

gives:
```
root[1] (1:1-2:6, 0-11)
└─ element[2] (1:1-2:6, 0-11) [tagName="div"][properties={"class":"align-right"}]
   ├─ element[1] (1:1-1:2, 0-1) [tagName="p"]
   │  └─ text: "a" (1:1-1:2, 0-1)
   └─ element[1] (3:1-3:2, -1--1) [tagName="p"]
      └─ text: "b" (3:1-3:2, -1--1)
```

Instead of one align-right and one align-center block.

(I fixed this issue in #98 )
